modules/teacher_emails/professor_embedder.py

```diff
@@ -10,10 +10,11 @@ def gen_embed(self, profs: Iterable[Professor]) -> discord.Embed:
 		embed = discord.Embed(colour=discord.Colour.green())
 
 		for prof in profs:
-			embed.add_field(
-				name=prof.name,
-				value=f"> {prof.linked_emails()}\n> {prof.subjects}",
-				inline=False,
-			)
+			if prof.emails:
+				embed.add_field(
+					name=prof.name,
+					value=f"> {prof.linked_emails()}\n> {prof.subjects}",
```

Comment:
There's a block quote function I put in utils, but of course you don't have to use it for something simple like this.